packages/ui/src/pages/connect/login.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import { hot } from 'react-hot-loader'
+import styled, { css } from 'styled-components'
+import { Icon } from '@mdi/react'
+import { mdiGoogle } from '@mdi/js'
+
+import { useUser } from '@/providers/user'
+import { useLoginConnectionsQuery } from '@/graphql/generated'
+
+const Container = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`
+
+const Section = styled.div`
```

Comment:
Section and container are the same for login and failed. Can it be shared or they are very specific for this pages?

Reply:
Its just placeholder stuff, its fine